src/useFormState.ts

```diff
@@ -371,7 +381,7 @@ export default function useFormState<T>(
     ),
     onLayout: layout(k, h),
     onBlur: blur(k, h),
-    value: (values?.[k] || '') as string,
+    value: objectPath.get(values as any, k as string),
```

Comment:
We should probably create a function getValue(values, k) with logic in it if k contains '.' use objectPath.get() otherwise use normal